tests/python/relay/test_type_infer.py

```diff
@@ -416,6 +416,21 @@ def test_dynamic_function():
     assert mod["main"].params[0].checked_type == s_tt
 
 
+@pytest.mark.xfail(raises=tvm.error.TVMError)
```

Comment:
let's use `with pytest.raises` instead (https://docs.pytest.org/en/stable/assert.html#assertions-about-expected-exceptions), because the testcase tests for throwing an exception, not a known limitation

Reply:
Good to know, I was wondering what the right way to do that was